src/posts/czi-eoss-grant-conclusion/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Reflecting on Xarray’s first development grant"
+date: "2022-06-03"
+authors:
+  - Joe Hamman
+summary: "A brief recap of our CZI EOSS grant, acomplishments and learnings."
+---
+
+In early 2020, we submitted a [proposal](https://doi.org/10.6084/m9.figshare.12709556.v1) to the [Chan Zuckerberg Initiative’s (CZI) Essential Open Source Software for Science (EOSS)](https://chanzuckerberg.com/eoss/) grant program. The goal of the proposal was to accelerate progress on key parts of our [roadmap](https://docs.xarray.dev/en/v2022.03.0/roadmap.html), providing dedicated support to augment regular volunteer contributions, enabling work on issues that are often too large to tackle by through volunteer-only contributions. The proposal was funded in late 2020 and our team spent the last year and a half working on a handful of new features and complex internal refactors. Having just closed out the grant, this blog post takes a look at what we proposed and how the development turned out.
+
+## Community Engagement
+
+More than any single feature addition, we felt like a key need for the Xarray project was a concerted focus on understanding and supporting our user community. To this end, we took on three tasks:
+
+1. [Anderson Banihirwe](https://github.com/andersy005) developed a new project website: [xarray.dev](https://xarray.dev) and [blog](https://xarray.dev/blog). This site includes a high-level introduction to Xarray, an interactive demo using [Pyodide](https://pyodide.org/en/stable/), and pointers to other cross-project resources. We also moved Xarray’s documentation site to a new URL: [docs.xarray.dev](https://docs.xarray.dev).
+2. We made numerous improvements to Xarray’s core documentation and extended [tutorial](https://tutorial.xarray.dev/intro.html). Notably, we reworked our [examples gallery](https://docs.xarray.dev/en/stable/gallery.html) and [tutorial page](https://docs.xarray.dev/en/stable/tutorials-and-videos.html).
```

Comment:
@andersy005 - please review and suggest any edits.

Reply:
apart from @dcherian's suggestion below, this section looks good.